Makefile.am

```diff
@@ -49,7 +49,7 @@ if WITH_TRUST_MODULE
 include trust/Makefile.am
 endif
 
-SUBDIRS = . doc po
+SUBDIRS = . doc po bash-completion
```

Comment:
We use automake in a non-recursive way if possible:
https://www.gnu.org/software/automake/manual/html_node/Alternative.html#Alternative

Reply:
Thank you for the suggestion. I will try to rewrite it to include.